### PR TITLE
2.10.4-RC2

### PR DESCRIPTION
**Pull request by retronym**
Generates: https://github.com/scala/scala-lang/pull/156

Review by @adriaanm

### CODE REVIEW

File: hand-written.md [hunk 1]
```diff
@@ -2,24 +2,27 @@
...
  - create new jira filters for fixed bugs, known issues,
  - link to closed milestone (after clearing milestone of un-merged PRs with https://github.com/adriaanm/binfu/blob/master/hubfu.sh#L29)
- - versions to search-replace: 2.10.4-RC1 (current release), 2.10.3 (previous release)
+ - versions to search-replace: 
+    - 2.10.4-RC2 (current release), 
+    - s/second/third/
+    - 2.10.3 (previous release)
 -->
-We are very happy to announce the first release candidate of Scala 2.10.4!
+We are very happy to announce the third release candidate of Scala 2.10.4!
```
Comment (adriaanm) [inline hand-written.md]: second? :)

Comment (retronym) [inline hand-written.md]: Darn. I just about made it "next".

Comment (adriaanm) [inline hand-written.md]: forSome? just `_`?

**Comment by adriaanm** [comment]
LGTM

**Comment by adriaanm** [comment]
I took a slightly different approach in master's hand-written: avoid saved jira filters and use links with embedded queries instead, so that we can search/replace version numbers in the urls, or rather, do that automatically.

**Comment by retronym** [comment]
Sounds much better!

**Pull request merged**